test/functional/admin/generic_editions_controller_tests/political_document_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+class Admin::GenericEditionsController::PolticalDocumentsTest < ActionController::TestCase
+  tests Admin::NewsArticlesController
+
+  setup do
+    login_as :policy_writer
+  end
+
+  view_test "displays the political checkbox for privileged users " do
+    login_as :managing_editor
+    edition = create(:news_article, first_published_at: 2.days.ago)
```

Comment:
These tests will want updating too: You'll need to create a published edition and then a subsequent draft from that to simulate an actual edition that has been previously published. Something like this:
```
published_edition = create(:published_news_article)
new_draft = create(:news_article, document: published_edition.document)
```